.github/workflows/tsp-client-tests.yml

```
@@ -0,0 +1,55 @@
+name: tsp-client - Test
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    paths:
+      - tools/tsp-client/**
+
+jobs:
+  tsp-client:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+        node-version: [18.19, 20]
```

```suggestion
        node-version: [18, 20]
```

I think you should be able to just use 18, it should install 18.latest (or close to latest) by default.